src/assert.rs

```diff
@@ -25,11 +39,13 @@ impl default::Default for Assert {
     ///
     /// Defaults to asserting _successful_ execution.
     fn default() -> Self {
+        let cargo_path = find_cargo();
```

Comment:
This fn can currently panic. We should deal with the result instead and maybe cache the error in the Assert struct so we can return it only when running execute or unwrap

Reply:
The main value of caching the result would be for when calling `.execute()` rather than `.unwrap()`, right?

Also, should the result of `find_cargo` be shoved off into a `lazy_static!`?